### PR TITLE
Remove hard-coded libraries part 1

### PR DESCRIPTION
**Pull request by JYC333**
Remove part of hard-coded libraries
- eslint
- jquery.mark.es6
- jsplumb
- knockout.js
- normalize.min.css
- wheel-zoom.min.js
- fancytree
- bootstrap

`bootstrap` can be updated to v5 later

Related TriliumNext/trilium#5075

### CODE REVIEW

File: src/routes/assets.ts [hunk 1]
```diff
@@ -70,11 +70,30 @@ function register(app: express.Application) {
...
 
     app.use(`/${assetPath}/node_modules/panzoom/dist/`, persistentCacheStatic(path.join(srcRoot, '..', 'node_modules/panzoom/dist/')));
-
     // i18n
     app.use(`/${assetPath}/node_modules/i18next/`, persistentCacheStatic(path.join(srcRoot, "..", 'node_modules/i18next/')));
     app.use(`/${assetPath}/node_modules/i18next-http-backend/`, persistentCacheStatic(path.join(srcRoot, "..", 'node_modules/i18next-http-backend/')));
     app.use(`/${assetPath}/translations/`, persistentCacheStatic(path.join(srcRoot, "public", "translations/")));
+
+    // Seems work without this, not sure will cause other bugs. Maybe can test without this for a while.
+    // app.use(`/${assetPath}/node_modules/eslint/bin/`, persistentCacheStatic(path.join(srcRoot, '..', 'node_modules/eslint/bin/')));
```
Comment (eliandoran) [inline src/routes/assets.ts:79]: ESLint is required for linting frontend and backend script notes, without it JavaScript errors are not displayed and there is an error in the console:

image

**Review by eliandoran — CHANGES_REQUESTED** [review]
Overall seems quite good.

**Comment by JYC333** [comment]
It causes some errors after I add eslint path into assets, but it fixed if I comment out eslint in `libraries/codemirror/addon/lint/eslint.js`. Will remove this also in this PR.
image

File: src/routes/assets.ts (hunk 1 above)
Comment (JYC333) [reply src/routes/assets.ts]: It causes some errors after I add eslint path into assets, but it fixed if I comment out eslint in `libraries/codemirror/addon/lint/eslint.js`. Will remove this also in this PR.
image

**Review by eliandoran — APPROVED** [review]

**Pull request merged**